Comments added to MainActivity.java

shamathmika opened the pull request:
Added comments to MAinActivity.java, changed comments to standards in all files and removed unnecessary import statements.

shamathmika committed:
Comments added to MainActivity.java

Changed comments to standards in all files and removed unnecessary
import statements.

sandarumk commented:
https://github.com/systers/PCSA-Android/wiki/PR-Best-Practices
Read the wiki.
The commit message is wrong. You have not only done changes to MainActivity.

shamathmika commented:
@sandarumk 
Sorry about that. New here. Will make the changes and send a PR again. Thanks!

sandarumk commented:
I haven't finished my review yet. If you can wait until it is finished, it will avoid you the trouble of changing the PR once again.

shamathmika commented:
@sandarumk 
Sure.

sandarumk commented:
There are certain files in a project like .properties file which shouldn't be committed. You have to remove them from your commit too.

sandarumk commented:
Please do the changes in the remaining classes only. We can leave the other modules to other new contributors. In that way we can limit to one commit for the comments in remaining classes.

sandarumk commented:
Please do the changes in the remaining classes only. We can leave the other modules to other new contributors. In that way we can limit to one commit for the comments in remaining classes. Can you make a new issue for removing unused imports. So we can have a separate PR and a commit for that which makes this clean.

shamathmika commented:
@sandarumk 
Okay. Thank you :) I'll make the changes and send a PR asap.

shamathmika committed:
Revert "Comments added to MainActivity.java"

This reverts commit 8cbf936fc525c15ce9c0ed84ff33c566f2372a66.

shamathmika committed:
Added class and method comments to MainActivity.java

shamathmika committed:
Added .iml file to .gitignore

shamathmika committed:
Added .iml to .gitignore

shamathmika commented:
@sandarumk 
Is this right?
https://github.com/systers/PCSA-Android/pull/281/commits/5e63937537888de760dc5cbf8cad80bb1c6354ba

sandarumk commented:
Did you go through all the remaining classes? Only the MainActivity lacks comments?

shamathmika commented:
@sandarumk 
Yes. All the classes not included in any sub-modules have comments.

sandarumk commented:
I found that the classes like MainActivityFragment and Constants do not have comments in the standards way. (Those were the only classes I went through after reading your comment.) Please go through the remaining classes again and add comments.

shamathmika committed:
Standardised class comments and added method comments

Added author and since fields in the class comments in Constants.java
and MainActivityFragment.java.
Added method comments to NavDrawerListAdapter.java

shamathmika committed:
ignore

shamathmika commented:
@sandarumk 
Hey, sorry! They had comments, so I thought it was expected to let them be.
Anyway, made the changes.
Is this it? 
8ad3ac700959afd43f4330d26994c79a8231eb7e

sandarumk commented:
1. Too much commits
2. Your PR still contains the app.iml file

sandarumk commented on app/src/main/java/com/peacecorps/pcsa/MainActivity.java:
```diff
@@ -46,6 +46,14 @@
 import java.util.List;
 
+/**
+ * Activity used to load the right placeholder fragment based on availability of instance from
```
Comment too long.

sandarumk reviewed (commented):
Too much commits  and till contains the app.iml file

sandarumk commented:
@shamathmika Any update on this?

shamathmika commented:
@sandarumk 
Hi! I am extremely sorry. Is it okay if I do this after December 5th?

sandarumk commented:
Sure.

sandarumk commented:
@shamathmika any updates?

shamathmika commented:
@sandarumk hi. I am having too many troubles with committing.

sandarumk commented:
What are the specific problems you are having?
You can also join systers slack group and ask your questions in #firstaide channel. We would be able to help you there. Just for a quick suggestion, have you tried to have a fresh copy and apply your changes?

top-gun007 commented:
@shamathmika rebasing might correct your merge conflict problems. Perhaps @sandarumk can throw more light on it.

codingblazer commented:
Hey @top-gun007 , Just rebasing won't resolve merge conflicts.
@shamathmika  Fetch upstream and then rebase, you will find all the Conflicting statements in your Android studio. Decide on what to keep (without erasing anybody else's work) . Then add Main Activity and continue with rebase. This will do it. :-)

top-gun007 commented:
Thats what I meant @codingblazer. Thank you for elaborating it.

shamathmika commented:
Hey!
Sorry for the late reply, I did the whole procedure again.
Is this good?
4107e534acc6e491a142baf7bafe9a5b9253d71a

top-gun007 commented:
The following points might help:

- reference your issue in the pr so that its easy for the mentors to know regarding what issue the pr was.

- squash your commits if they belong to same issue unless stated otherwise. use (git rebase -i yourbranch~n).

- this one has merge conflicts resolve them. A bit of google search might do it.

-   remove the necessary files as suggested above(app.iml etc.)

sandarumk commented:
@shamathmika You should go with @top-gun007's suggestions.

sandarumk commented:
@shamathmika are you still on this? Of you want to start this fresh, you can close this PR and send a new one.

top-gun007 commented:
@sandarumk I think she might have deleted the repo corresponding to this PR thats why it is showing "Unknown repository" at the top. If she is not working on it then I would like to correct it.

shamathmika commented:
@sandarumk I had sent another PR. I don't think that had any problems. I had even shared it here (4107e53). If that is not right, I think you should let someone else do it since I'm taking way too long figuring it out.

sandarumk commented:
@shamathmika first of all there is no other PR from you. This is the only PR I found from you in the list. What you have mentioned here is the commit. And yes your mentioned commit has no problems.  This PR contains https://github.com/systers/FirstAide-Android/pull/281/files these changes. 
There is no hurry of you really want to do this. What you can do is fetch and rebase your fork, make a new branch and apply your latest commit by cherrypicking. Then send a PR to systers/develop from that branch. 
You can do a basic google search for fetch, rebase and cherrypicking. This is the easiest solution I can think of. (Some other more easier solutions can exist).

sandarumk commented:
If you find cherrypicking hard, just copy your comments in your new branch. Do not copy the whole file. It might have updated from your existing copy.

sandarumk commented:
Closing this currently since no response. If you are still on it, pease send a new PR
